### PR TITLE
CORDA-2404: Highlight resolution for the use of V3 confidential-identities

### PR DESCRIPTION
**Pull request by shamsasari**
This is a follow-up to https://github.com/corda/corda/pull/4536.

### CODE REVIEW

File: docs/source/release-notes.rst [hunk 1]
```diff
@@ -287,6 +287,20 @@ The TLS_DHE_RSA_WITH_AES_128_GCM_SHA256 family of ciphers is retired from the li
...
+Confidential identities
++++++++++++++++++++++++
+
+If any of your CorDapps are using the experimental confidential-identities module and have not been updated to use the new API
+(see :ref:`the upgrade notes<update_swap_ident_ref>`) then you will also need to load a patched V3 confidential-identities CorDapp jar.
+Failing this, the flows in those CorDapps may fail with the error::
+
+  java.lang.NoClassDefFoundError: net/corda/confidential/SwapIdentitiesFlow
+
+If this is the case please contact R3 Support.
```
Comment (shamsasari) [inline docs/source/release-notes.rst]: Is this OK to say @gendal ?

**Comment by shamsasari** [comment]
Actually I think I have a better solution. Closing for now.

**Comment by gendal** [comment]
wow... one of those things where if you don't get to it in time it ceases to be urgent! sorry for not looking at it earlier

**Comment by shamsasari** [comment]
No, I'm glad you didn't!

**Pull request closed without merging**